pubsub/subscription.go

```diff
@@ -210,14 +210,90 @@ func (oidcToken *OIDCToken) toProto() *pb.PushConfig_OidcToken_ {
 	}
 }
 
+// BigQueryConfigState denotes the possible states for a BigQuery Subscription.
+type BigQueryConfigState int
```

Comment:
We added a SubscriptionState field as well as part of this feature. Should we add that here as well?

Reply:
Added. I also made the fake return `Active` for both Subscription/BigQueryConfigState by default.